server/task_executor.go

```diff
@@ -169,6 +169,19 @@ func postMigration(ctx context.Context, server *Server, task *api.Task, vcsPushE
 		}
 	}
 
+	// Remove schema drift anomalies.
```

Comment:
Also, shall we do it after we do the write-back? This should really be the fail-open, best-effort thing to do.